eng/Analyzers.targets

```
@@ -14,7 +14,8 @@
   <PropertyGroup Condition="'$(RunAnalyzers)' != 'false'">
     <EnableSingleFileAnalyzer Condition="
       '$(EnableSingleFileAnalyzer)' == '' And
-      '$(TargetFrameworkIdentifier)' == '.NETCoreApp'">true</EnableSingleFileAnalyzer>
+      '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
```

nit: I know this was already there before but please use lowercase conditional operators. Just style. I would really like msbuild to add a style analyzer so that we don't have the inconsistencies in our repo.

```suggestion
      '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
```